examples/lbfgs/driver.py

```diff
@@ -73,27 +70,29 @@ def set_weights(new_weights):
   def net_reinitialization(net_vars):
     return net_vars
 
-  # Create a reusable variable for the network.
+  # Register the network with Ray and create a reusable variable for it.
+  ray.register_class(LinearModel)
   ray.reusables.net_vars = ray.Reusable(net_initialization, net_reinitialization)
 
   # Load the weights into the network.
   def load_weights(theta):
```

Comment:
can you define a generic function in tfutils.py which takes a dict and flattens it into a numpy array? It might be necessary to go from dicts to ordered dicts for that (and do ray.register_class(ordereddict) or similar)